backend/src/services/bambora.js

```diff
@@ -22,11 +22,16 @@ module.exports = {
     },
 
     createBamboraPaymentRequest: (paymentModel) => {
+        const amount = paymentModel.amount * 100;
```

Comment:
Why do you multiply it to 100?

Reply:
This is to resolve an issue with the Bambora payment. It needs to be in this format. 
100 = 1
1000 = 10
and likewise.